lib/_http_outgoing.js

```diff
@@ -637,7 +638,7 @@ OutgoingMessage.prototype.addTrailers = function addTrailers(headers) {
     if (typeof field !== 'string' || !field || !checkIsHttpToken(field)) {
       throw new ERR_INVALID_HTTP_TOKEN('Trailer name', field);
     }
-    if (checkInvalidHeaderChar(value)) {
+    if (checkInvalidHeaderChar(value) !== null) {
```

Comment:
The addition of the `!== null` here shouldn't be necessary since the function will return falsy if the check passes